.travis.yml

```diff
@@ -0,0 +1,21 @@
+script:
+  - pushd source
+  - make -j2
+  - popd
+
+sudo: false
+
+addons:
+  apt:
+    packages:
+    - latexmk
```

Comment:
Because I started by copying the list of required packages from the README.

We could set up a build matrix and have it build using `latexmk` and with the `Makefile` to exercise both build options, too, I s'pose.

Installing the `latexmk` package isn't terribly time-consuming or onerous in any case, though.